index.html

```diff
@@ -19,23 +19,170 @@
   </header>
 
   <section class="main-container">
-    <section class="corner-image">
-        <div class="container">
-            <h1 class="logo-message">I'm  Dan</h1>
-            <h3 class="dev-intro">I'am a Software Developer</h3>
-            <p class="background">
-                I can help you build a product , feature or website Look through some of my work and experience!
-                If you like what you see and have a project you need coded, do not hestiate to contact me.
-            </p>
-        </div>
-        <ul class="social-media  container">
-            <li><a href="https://github.com/dunnyk"><img src="image/github.png" alt="github icon"></a></li>
-            <li><a href="https://www.linkedin.com/in/daniel-njoroge-8091b3128/"><img src="image/linkedin.png" alt="linkedin icon"></a></li>
-            <li><a href="https://twitter.com/dankama231"><img src="image/twitter.png" alt="twitter icon"></a></li>
-            <li><a href="https://www.instagram.com/danielkamau231/"><img src="image/finger.png" alt="peace finger sign"></a></li>
-            <li><a href="https://medium.com/@danielkamau231"><img src="image/medium.png" alt="M logo"></a></li>
+    <div class="container">
+        <h1 class="logo-message">I'm  Dan</h1>
+        <h3 class="dev-intro">I'am a Software Developer</h3>
+        <p class="background">
+            I can help you build a product , feature or website Look through some of my work and experience!
+            If you like what you see and have a project you need coded, do not hestiate to contact me.
+        </p>
+    </div>
+    <ul class="social-media  container">
+        <li><a href="https://github.com/dunnyk"><img src="image/github.png" alt="github icon"></a></li>
+        <li><a href="https://www.linkedin.com/in/daniel-njoroge-8091b3128/"><img src="image/linkedin.png" alt="linkedin icon"></a></li>
+        <li><a href="https://twitter.com/dankama231"><img src="image/twitter.png" alt="twitter icon"></a></li>
+        <li><a href="https://www.instagram.com/danielkamau231/"><img src="image/finger.png" alt="peace finger sign"></a></li>
+        <li><a href="https://medium.com/@danielkamau231"><img src="image/medium.png" alt="M logo"></a></li>
+    </ul>
+  </section>
+
+
+  <!-- WORK SECTION -->
+
+  <section class="main-wrap">
+    <article class="work-section">
+      <div class="work-title">
+        <h1>My Recent Works</h1><br>
```

Comment:
- [ ] Kindly change this h1 tag to any hx tag from h2- h6, it is better for your page to have only one h1 tag as the main heading of the page, this will make your project more semantic and easily readable by screen readers 👍🏿